src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Azure/Functions/IDefaultHttpRequest.cs

```diff
@@ -0,0 +1,35 @@
+// <copyright file="IDefaultHttpRequest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Azure.Functions
+{
+    /// <summary>
+    /// Interface for ducktyping
+    /// </summary>
+    public interface IDefaultHttpRequest
```

Comment:
Are these properties used anywhere? (asking because they are all `object`)
If not, we should remove them to avoid the expense of mapping them.